visualization/app/codeCharta/codeMap/rendering/geometryGenerator.ts

```diff
@@ -58,11 +58,17 @@ export class geometryGenerator {
             y : n.z0,
             z : n.y0,
             width : n.width,
-            height : Math.max(n.height, 5.0),
+            height : this.ensureMinHeightIfDeltaNotNegative(n.height, n.heightDelta),
             depth : n.length
         };
     }
 
+    private ensureMinHeightIfDeltaNotNegative(x: number, d: number): number {
+        if(d >= 0) {
+            return Math.max(x, 5.0);
```

Comment:
Where comes this magic value 5? Maybe some kind of configuration of CC? Is there a coupling to another magic value?